public/app/ui/components/search/results/search-results.html

```diff
@@ -25,7 +25,7 @@
                     <thead>
                     <tr>
                         <th>#</th>
-                        <!-- ko foreach: { data: fields, as: 'field' } -->
+                        <!-- ko foreach: { data: resultFields, as: 'field' } -->
```

Comment:
I'm not really sure how this could have happened.  The field changed from `searchResultFields` to `resultFields`, there was never anything called `fields`, and it was `searchResultFields` in `table-results.html` before that was merged into `search-results.html`.  Also I tested it and it was working after that change.

In any case this change looks correct.